db/active_replicator_push.go

```diff
@@ -341,3 +341,15 @@ func (apr *ActivePushReplicator) _startPushNonCollection() error {
 
 	return nil
 }
+
+func (apr *ActivePushReplicator) incrementHitandMissStatsCollections(collectionID *int, since SequenceID) {
+	for _, v := range apr.namedCollections {
+		if v.collectionIdx == collectionID {
+			if !since.IsNonZero() {
+				v.Checkpointer.stats.GetCheckpointMissCount++
+			} else {
+				v.Checkpointer.stats.GetCheckpointHitCount++
+			}
+		}
+	}
+}
```

Comment:
We could move this method onto `ActiveReplicatorCommon` (which is embedded inside `apr` - instead of having a separate copy for push and pull.